Server/src/services/tools/manage_tools.py

```
@@ -0,0 +1,142 @@
+"""
+manage_tools – server-only meta-tool for dynamic tool group activation.
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)